pkgs/development/python-modules/dask/default.nix

```diff
@@ -43,8 +38,20 @@ buildPythonPackage rec {
     pandas
     partd
     toolz
+  ] ++ lib.optionals withExtraComplete [
+    distributed
```

Comment:
What does this do? Just add it as additional runtime dependency, or does it have an impact on the build? If it is the former, then it should be removed: optional dependencies we don't do anything with; users can just add the extra dependency with `withPackages`. If it is the latter, then having the option is fine, it just cannot be used in `python-packages.nix` as a run-time dependency.